cube/cube.cpp

```diff
@@ -3394,35 +3733,84 @@ int main(int argc, char **argv) {
 
     demo.init(argc, argv);
 
+    switch (demo.wsi_platform) {
+        default:
```

Comment:
This switch needs a case for `VK_USE_PLATFORM_DISPLAY_KHR`, otherwise when `--wsi DISPLAY` is set, it falls back to the default and exits. I think nothing special needs to be done so it can just `break`. Also on cpp.